test/libsolidity/syntaxTests/comments/multiline_unicode_direction_override_7.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+    function f() public pure {
+        /* LRO‭ LRE‪ RLE ‫  PDF‬ RLO‮ PDF ‬ PDF‬
```

Comment:
I wanted to point out the FDP and only then realized what's going on. It works :) Lost opportunity for O RLY though :)